jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,238 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import asyncio\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "from systemlink.clients.nitdmreader.api.metadata_api import MetadataApi\n",
+    "from systemlink.clients.nitdmreader.models.file_list import FileList\n",
```

Comment:
These two imports can be removed and all occurrences of `MetadataApi` and `FileList` must be prefixed with `tdmreader`.

Reply:
ok